.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchDateTimePeriodParserConfiguration.cs

```diff
@@ -179,21 +179,21 @@ public bool GetMatchedTimeRange(string text, out string timeStr, out int beginHo
             return true;
         }
 
-        public int GetSwiftPrefix(string text)
-        {
-            var trimmedText = text.Trim();
-
-            var swift = 0;
-            if (FutureRegex.IsMatch(trimmedText))
-            {
-                swift = 1;
-            }
-            else if (PreviousPrefixRegex.IsMatch(trimmedText))
-            {
-                swift = -1;
-            }
-
-            return swift;
+        public int GetSwiftPrefix(string text)
```

Comment:
What happened here? Are these line ending changes? Can you revert them? It seems they are spread through many files.